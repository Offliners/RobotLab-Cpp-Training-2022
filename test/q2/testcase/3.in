154
deceAse
WhaLE
resPeCt
OVERCOmE
GEOMETRY
APPLIANCE
IrResPonSIbLE
peRmissIon
BAttle
nEGative
sKELEtON
inDepEndenCe
nonetheless
GENEROSITY
altHough
iNtERact
aSSAULt
click
saTuRdaY
iNJECT
ACcUMuLAte
grin
committee
CLOUD
sHAmpoo
cunnINg
LocK
TrAfFiC
than
ShIP
bLOod
kiss
comment
CuP
experTise
foRth
resTRoom
secUrity
APPlicAtiOn
ColoRFUl
boil
ouTwArd
PoiNT
NuCLear
dEPARTUrE
heiGhtEn
ETHic
HINT
wEAlTH
MEtHOD
snoW
vIolin
PrIvIlEGE
destroy
Hardware
JunE
caRpet
austrAlia
goodness
waIst
doom
beNEatH
HAPPiNESS
ACREAGE
nOISE
OUTdooRS
STATURE
attitude
affiliated
ridDle
exPorT
deLIciOUs
inTELlECT
PERFUME
iNFlATION
unRest
CUT
LongevitY
coVeR
OVeRLook
bulletin
autonomy
pEAr
TERRIFY
eXPOSUrE
surviVal
cRaFt
tRActOr
roUgH
Copy
pUrSuE
EDITIOn
SIxTy
ELECTRICIAN
insect
size
OBjeCT
comeR
MELT
comPLIMe0t
aGOny
NARRATE
fry
fevEr
CABin
spot.
absEnt
coNsolidAtE
acclaim
MiXTURE
lARGE
mare
FIU
muddy
MARK
39550u62o7
devicE
mAGNEtIC
meEt
SolDier
dedicate
NUDe
likELihOoD
meaN
PoLlUTioN
EVERY
SecuRe
rescue
CuCuMBeR
port
VICIOUS
anGel
COUNTY
FAMILIAR
HEAR
cOverage
DoNe
dIsCuSS
DISPOSAL
RAy
context
Peach
esTImATE
bAm
gIve
VACUUM
INVESt
SubmARINE
CIRCLe
upStairs
gallery
dESeRT
compact
ToNGUE
