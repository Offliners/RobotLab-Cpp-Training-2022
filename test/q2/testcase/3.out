n. 死亡
n. 鯨
n./v. 尊敬
v. 克服
n. 幾何學
n. 器具；裝備
adj. 不須負責任的
n. 准許
n./v. 戰役
adj./n. 否定的
n. 骨骸
n. 獨立
adv./conj. 但是；仍然
n. 慷慨
conj. 雖然；然而
v. 互動
n. 攻擊
v. 使發出聲
n. 星期六
v. 注射
v.  累積
v./n. 露齒而笑
n. 委員會
n./v. 雲；遮蔽
n. 洗髮(精)
adj. 狡猾的；奸詐的
n./v. 鎖
n. 交通
prep./conj. 比；比較
n. 船
n. 血液
n./v. 吻
n./v. 評論
nJv. 杯子
n. 專業知識
adv. 向前
n. 廁所
n. 安全
n. 應用；申請書
adj. 多采多姿的
v./n. 沸騰
adj. 外面的；肉體的
n./v. 尖端；持向
adj. 核子的
n. 分離；啟程
v. 增高
n. 倫理
n./v. 暗示
n. 財富
n. 方法
n./v. 雪
n. 小提琴
n. 特權
v. 摧毀
n. 五金器具；硬體
n. 六月
n. 地毯
prop./n. 澳洲
n. 善良
n. 腰
n./v. 厄運；注定
prep. 在…下面
n. 快樂
n. 英畝數
n. 噪音
adv. 在戶外
n. 高水平；身材
n. 態度
adj. 相關聯的
n. 謎題
v./n. 出口
adj. 美味可口的
n. 理解力
n. 香味；香水
n. 通貨膨脹
n. 動亂
n./v. 切
n. 長壽
n./v. 封面；遮蓋
v. 俯視
n. 公報；新聞快報
n. 自治(權)
n. 梨
v. 使驚恐
n. 暴露；陳列
n. 殘存
n. 工藝；太空船
n. 拖拉機
adj. 不平的；粗魯的
n./v. 副本；拷貝
v. 追捕
n. 發行數；版本
adj./n. 六十
n. 電工；電氣技師
n. 昆蟲
n. 尺寸
n. 物體；目標
n. 角落
v. 融化
Unknown
n. 痛苦
v. 敘述
v. 油炸
n. 發燒；狂熱
n. 客艙
n./v. 地點；散布
adj. 缺席的
v. 鞏固
v./n. 歡呼；歡呼聲
n. 混合物
adj. 大的
n. 母馬；母驢
n. 流行性感冒
adj. 泥的；骯髒的
n./v. 痕跡；記號
Unknown
n. 裝置
adj. 磁鐵的；帶磁性的
v. 遇見
n. 軍人
v. 奉獻
adj. 裸體的
n. 可能性
adj. 吝嗇的
n. 污染
adj. 每個
adj. 安全的
n./v. 救助
n. 黃瓜
n. 港
adj. 惡意的
n. 天使
n. 郡；縣
adj. 熟悉的
v. 聽見
n. 報導；某事所包含的範圍
adj. 完成的
v. 討論
n. 處理；配置
n. 光線
n. 上下文
n. 桃子
n./v. 估計；預計
n. 穀倉；牲口棚
v. 給；送給
n. 真空；空虛(感)
v.(n. )投資；投入
n. 潛水艇
n./v. 循環；畫圓圈
adj. 在樓上
n. 畫廊；美術館
n. 沙漠
adj. 緊密的
n. 舌(頭)
